requirements/test.txt

```diff
@@ -1,28 +1,34 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
 #    make upgrade
 #
-amqp==5.1.1
+amqp==5.2.0
     # via
     #   -r requirements/base.txt
     #   kombu
-asgiref==3.6.0
+asgiref==3.7.2
     # via
     #   -r requirements/base.txt
     #   django
-astroid==2.11.7
+astroid==2.13.5
     # via pylint
-billiard==3.6.4.0
+backports-zoneinfo[tzdata]==0.2.1;python_version<"3.9"
```

Comment:
**This change was applied manually:** `backports.zoneinfo` has no support for later 3.8 Python versions.